tensorflow/contrib/tensorrt/segment/segment.cc

```diff
@@ -406,22 +406,43 @@ tensorflow::Status SegmentGraph(
   // Use a union-find to collect the nodes that belong to the same
   // segment. A node value of nullptr indicates that the node is not a candidate
   // for TRT.
+  std::unordered_set<string> unsupported_ops;
+  int num_unsupported_ops = 0;
   std::vector<UnionFind<SimpleNode*>> node_segments;
   for (int i = 0; i < graph->num_node_ids(); ++i) {
     SimpleNode* node = graph->FindNodeId(i);
     if (options.exclude_node_list.count(node->name()) != 0) {
-      VLOG(1) << "Not a TF-TRT candidate: " << node->name()
-              << " (excluded by segmenter option).";
+      VLOG(1) << "Not a TF-TRT candidate, "
+              << "(Op type: " << node->tf_node()->type_string() << "), "
+              << "(Op name: " << node->name() << "), "
+              << "(Reason: excluded by segmenter option)";
+      unsupported_ops.emplace(node->tf_node()->type_string());
+      num_unsupported_ops++;
       node = nullptr;
     } else {
       const Status status = candidate_fn(node->tf_node());
       if (!status.ok()) {
-        VLOG(1) << "Not a TF-TRT candidate: " << node->name() << ": " << status;
+        VLOG(1) << "Not a TF-TRT candidate, "
+                << "(Op type: " << node->tf_node()->type_string() << "), "
+                << "(Op name: " << node->name() << "), "
+                << "(Reason: " << status << ")";
+        unsupported_ops.emplace(node->tf_node()->type_string());
+        num_unsupported_ops++;
         node = nullptr;
       }
     }
     node_segments.emplace_back(node);
   }
+  string msg = "There are "
+               + std::to_string(num_unsupported_ops)
+               + " unsupported ops of "
+               + std::to_string(unsupported_ops.size())
+               + " different types in the graph: ";
+  for (const auto& elem: unsupported_ops) {
+    msg += elem + ", ";
+  }
+  LOG(INFO) << msg << "(For more information see "
```

Comment:
This may make people confused, since some of those op types are not really unsupported, but may be excluded by the segment options, or some other reasons (e.g. Reshape op is supported but only for the ones that doesn't change the batch dimension). What do you think?

Reply:
You are right. We need to change the wording a bit. How about this: 

"There are 6 ops of 5 different types in the graph that are not converted to TRT: ArgMax, NoOp, Placeholder, Squeeze, Identity, (For more information see https://docs.nvidia.com/deeplearning/dgx/integrate-tf-trt/index.html#support-ops)."

Reply:
Done